### PR TITLE
Rebuild coreutils-5.0 with tcc+musl and more utilities

### PR DESCRIPTION
**Pull request by pder**
Add expr, date, dd, sort, uname, uniq

### CODE REVIEW

File: sysa/coreutils-5.0/mk/pass2.mk [hunk 1]
```diff
@@ -0,0 +1,121 @@
+PACKAGE=coreutils
```
Comment (stikonas) [inline sysa/coreutils-5.0/mk/pass2.mk:1]:
```suggestion
# SPDX-FileCopyrightText: 2021 Paul Dersey <pdersey@gmail.com>
# SPDX-FileCopyrightText: 2021 Andrius Štikonas <andrius@stikonas.eu>
#
# SPDX-License-Identifier: GPL-3.0-or-later

PACKAGE=coreutils
```

**Review by stikonas — CHANGES_REQUESTED** [review]

File: sysa/coreutils-5.0/mk/pass2.mk (hunk 1 above)
Comment (fosslinux) [reply sysa/coreutils-5.0/mk/pass2.mk:1]: @pder coudl you please do this

**Comment by pder** [comment]
Rebased onto master, updated copyright info in pass2.mk, updated numbering in run.sh and README.md and added comm to coreutils build.

**Review by stikonas — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -325,3 +325,8 @@ be useful later for autotools.
 
 `diffutils` is useful for comparing two files. It is not immediately needed but
 is required later for autotools.
+
+#### Part 27: coreutils 5.0
+
+`coreutils` is rebuilt against musl.  Additional utilities are built including
+comm, expr, date, dd, sort, uname, and uniq.
```
Comment (stikonas) [inline README.md:332]: Maybe backtick binaries in README? e.g. comm -> `comm`?

**Review by stikonas — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (pder) [reply README.md]: Updated the README with backticks

**Comment by fosslinux** [comment]
Hm, that was not exactly my intention, but that's ok.

**Pull request merged**